modules/core-services/providers/token/dao.go

```diff
@@ -84,7 +91,10 @@ func (d *dao) QueryToken(ctx context.Context, req *pb.QueryTokensRequest) ([]tok
 }
 
 func (d *dao) CreateToken(ctx context.Context, req *pb.CreateTokenRequest) (*tokenstore.TokenStoreItem, error) {
-	obj := toModel(req)
+	obj, err := toModel(apis.GetUserID(ctx), req)
```

Comment:
Not good to directly get userId in `dao`, you should add Creator filed in `CreateTokenRequest `.

Reply:
done